src/weni/protobuf/connect/organization.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package organization;
```

Comment:
The package needs to contain the prefix `weni.connect.`  like in [project.proto](https://github.com/Ilhasoft/weni-protobuffers/blob/main/src/weni/protobuf/connect/project.proto#L3).
Remember to compile the protobuffers again.